### PR TITLE
NHD vars of interest

### LINKED ISSUE #50: Wieczorek variables of interest table target
There are many catchment attributes aggregated to the NHD network that are on sciencebase (e.g., https://www.sciencebase.gov/catalog/item/5cc89237e4b09b8c0b77f247). 

We need a target for the table that lists which of these variables we are interested in for inland salinity predictions.

### LINKED ISSUE #51: Decide and document how to spatially aggregate each Wieczorek variable of interest
Once we have the target for the Wieczorek variables of interest (#50), we need to decide and document how to spatially aggregate each of these variables from the NHD network to the PRMS network.

### PR DESCRIPTION
**Pull request by jsadler2**
Adding the csv for the NHD vars of interest. Included in this is a column `Aggretation_operation` which is the operation that should be performed to aggregate from the NHD to the PRMS network. 

This is what I came up with for a first cut at assigning the operations:

|operation| num variables applied to|
| --- | --- |
|`area_weighted_mean`|  168|
|`max`| 1|
|`min`|1|
|`sum`|7|

So almost all of them are `area_weighted_mean`.

The one that aren't `area_weighted_mean`:

image

closes #50, 
closes #51

### CODE REVIEW

**Comment by jds485** [comment]
Thanks! Your aggregation choices look good. I have only 2 that I think would be better as sum: NID_STORAGEYYYY and NORM_STORAGEYYYY to get total storage volumes in the aggregated catchment.

I'll note that the suggested aggregations are for the catchment values (CAT) instead of the divergence (ACC) or total (TOT) attributes. Maybe better to name the column `CAT_aggregation_operation`

File: 1_fetch/in/NHDVarsOfInterest.csv [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+Hydrologic Modifications,NORM_STORAGEYYYY,"The normal dam storage (in acre-feet) defined as the total storage space in all reservoirs in a flowline catchment that is below the normal retention level, including dead and inactive storage and excluding any flood control or surcharge storage. Value is based on dams built on or before YYYY, where YYYY is the last year for the decade of record (for example 1960 spans 1951 - 1960). The exceptions are 1930 and 2013 (1930 and before and 2010 to 2010 respectively).",CAT_NORM_STORAGEYYYY,ACC_NORM_STORAGEYYYY,TOT_NORM_STORAGEYYYY,acre-feet,https://www.sciencebase.gov/catalog/item/58c301f2e4b0f37a93ed915a,NA,NA,TRUE,area_weighted_mean
```
Comment (jsadler2) [inline 1_fetch/in/NHDVarsOfInterest.csv:99]:
```suggestion
Hydrologic Modifications,NID_STORAGEYYYY,"The maximum dam storage (in acre-feet) defined as the total storage space in all reservoirs in a flowline catchment below the maximum attainable water surface elevation, including any surcharge storage, of dams built on or before YYYY. Value is based on dams built on or before YYYY, where YYYY is the last year for the decade of record (for example 1960 spans 1951 - 1960). The exceptions are 1930 and 2013 (1930 and before and 2010 to 2010 respectively).",CAT_NID_STORAGEYYYY,ACC_NID_STORAGEYYYY,TOT_NID_STORAGEYYYY,acre-feet,https://www.sciencebase.gov/catalog/item/58c301f2e4b0f37a93ed915a,NA,NA,TRUE,sum
Hydrologic Modifications,NORM_STORAGEYYYY,"The normal dam storage (in acre-feet) defined as the total storage space in all reservoirs in a flowline catchment that is below the normal retention level, including dead and inactive storage and excluding any flood control or surcharge storage. Value is based on dams built on or before YYYY, where YYYY is the last year for the decade of record (for example 1960 spans 1951 - 1960). The exceptions are 1930 and 2013 (1930 and before and 2010 to 2010 respectively).",CAT_NORM_STORAGEYYYY,ACC_NORM_STORAGEYYYY,TOT_NORM_STORAGEYYYY,acre-feet,https://www.sciencebase.gov/catalog/item/58c301f2e4b0f37a93ed915a,NA,NA,TRUE,sum

```

**Comment by jsadler2** [comment]
Thanks for looking at this, @jds485. I've made the changes you suggested.

**Review by jds485 — APPROVED** [review]
Looks good to merge! Thanks!

**Pull request merged**